static/css/components/footer-locale.less

```diff
@@ -0,0 +1,55 @@
+.language-component__checkbox {
```

Comment:
Rename to language.less - footer-locale is an instance of this component and reusable

Reply:
Done.

Reply:
@jdlrobson strange how does renaming changes the bundlesize.
` FAIL  static/build/all.js: 128KB > maxSize 128KB (gzip) `

Reply:
#footer-locale is shorter than .language-component__checkbox and gzips better. Bundlesize is checking the size of the CSS file generated from this source code.

Reply:
We have id="footer-locale-menu" but so what changes do you want me to make.

Reply:
The code here is fine. I was responding to your comment about it being stranged that it impacted bundlesize and explaining why that happens.

Reply:
How should we fix this. Increase bundlesize?

Reply:
@jdlrobson, if we go ahead with JS dropdown with hover fallback, then we also need to update this accordingly. So should I update it now or wait for further discussion?